config/settings_ts/monitoring_groups.json

```diff
@@ -0,0 +1,126 @@
+{
+    "monitoring_groups": [
+        
+        {
+            "group_name": "salmonts_all",
+            "glue_jobs": [
+                {
+                    "name": "*-salmonts-*",
+                    "monitored_environment_name": "Dept1 Account [dev]",
+                    "sla_seconds": 0,
+                    "minimum_number_of_runs": 0
+                }
+            ]
+        },
+        {
+            "group_name": "salmonts_gluejobs_all",
+            "glue_jobs": [
+                {
+                    "name": "glue-salmonts-*",
+                    "monitored_environment_name": "Dept1 Account [dev]",
+                    "sla_seconds": 0,
+                    "minimum_number_of_runs": 0
+                }
+            ]
+        },
+        {
+            "group_name": "salmonts_sparkjobs",
+            "glue_jobs": [
+                {
+                    "name": "glue-salmonts-sparkjob-*",
+                    "monitored_environment_name": "Dept1 Account [dev]",
+                    "sla_seconds": 0,
+                    "minimum_number_of_runs": 0
+                }
+            ]
+        },
+        {
+            "group_name": "salmonts_pyjobs",
+            "glue_jobs": [
+                {
+                    "name": "glue-salmonts-pyjob-*",
+                    "monitored_environment_name": "Dept1 Account [dev]",
+                    "sla_seconds": 0,
+                    "minimum_number_of_runs": 0
+                }
+            ]
+        },
+        {
+            "group_name": "salmonts_lambdas",
+            "lambda_functions": [
+                {
+                    "name": "lambda-salmonts-*",
+                    "monitored_environment_name": "Dept1 Account [dev]",
+                    "minimum_number_of_runs": 0
+                }
+            ]
+        },
+        {
+            "group_name": "salmonts_stepfunctions",
```

Comment:
Initially, I thought about just having 3 monitoring groups:

1. for all python shell glue jobs
2. for glue workflow + pyspark jobs
3. step fun + lambdas.

Multiple groups here may become benefitial for throughout testing later, but now It might complicate things :)
Let's stick with just 3 monitored groups